src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1490,6 +1492,14 @@ def get_crash_data(crash_data, symbolize_flag=True):
           address_filter=lambda s: '0x' + s,
           reset=True)
 
+      # Libfuzzer deadly signal errors.
+      update_state_on_match(
+          LIBFUZZER_DEADLY_SIGNAL_REGEX,
+          line,
+          state,
+          new_type='deadly signal',
```

Comment:
Nit: capitalise 'deadly' to 'Deadly'.

Reply:
We should prefer to avoid spaces, use hyphen. Also, don't like the use of word deadly. maybe "Fatal-signal".

Reply:
qq: Are you testing this with a libFuzzer + ASan binary. Did you set ASAN_OPTIONS as handle_{sig*}=1. I was expecting this to be covered by the sanitizer instrumentation which mentions the details on deadly signal type, like Abrt, etc.

@kcc - wdyt ?

Reply:
I think this is an issue with the fuchsia asan integration. There's no real concept of signals, so asan isn't able to dump the stack automatically. I think the current implementation falls back to the libFuzzer stack dumper when e.g. __builtin_trap() is called. Maybe longer term we can make ASan handle the fake "signal", but for now this seems OK to unblock things. @nopsledder what do you think?

Reply:
Thanks for explaining. Just change to "Fatal-signal" and fine to keep this.